From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: aswin-mcw <aswin.kalies@multicorewareinc.com>
Date: Wed, 9 Sep 2026 13:14:49 +0530
Subject: [PATCH 067/109] Comment out AddVectoredContinueHandler in
 dll_crt0_0 to prevent potential issues during initialization

(cherry picked from commit c88dd7f90cc0170431410377e7e5f37999083634)
---
 winsup/cygwin/dcrt0.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/winsup/cygwin/dcrt0.cc b/winsup/cygwin/dcrt0.cc
index cb1251a72..4e2778742 100644
--- a/winsup/cygwin/dcrt0.cc
+++ b/winsup/cygwin/dcrt0.cc
@@ -779,7 +779,7 @@ dll_crt0_0 ()
     sigproc_init ();
 
   /* See comment preceeding myfault_altstack_handler in exception.cc. */
-  AddVectoredContinueHandler (0, myfault_altstack_handler);
+//   AddVectoredContinueHandler (0, myfault_altstack_handler);
 
   debug_printf ("finished dll_crt0_0 initialization");
 }
